x-pack/solutions/security/plugins/cloud_security_posture/public/pages/configurations/latest_findings/use_latest_findings_grouping.tsx

```diff
@@ -86,10 +86,10 @@ const getAggregationsByGroupField = (field: string): NamedAggregation[] => {
   ];
 
   switch (field) {
-    case FINDINGS_GROUPING_OPTIONS.RESOURCE_NAME:
+    case FINDINGS_GROUPING_OPTIONS.RESOURCE_ID:
       return [
         ...aggMetrics,
-        getTermAggregation('resourceName', 'resource.id'),
+        getTermAggregation('resourceName', 'resource.name'),
```

Comment:
can you provide some context around this change? wouldn't `resource.id` should be used in `FINDINGS_GROUPING_OPTIONS.RESOURCE_ID`?
just an observation as im not too familiar with this code

Reply:
I believe this is required to get more data for the group, as we use `resource.name` in the group title, we need to aggregate by it

Reply:
yes to what @maxcold said, this aggregation retrieves additional data used on this grouping visualization (.ie resources view will retrieve resources name so it can display the resource name under the resource id, cloud account view will retrieve the cloud provider and so on)